### PR TITLE
Fix pep8 errors in TUI source spoke

### PR DESCRIPTION
**Pull request by jkonecny12**

### CODE REVIEW

File: pyanaconda/ui/tui/spokes/installation_source.py [hunk 1]
```diff
@@ -426,7 +432,8 @@ def refresh(self, args=None):
         # check if there are any mountable devices
         if self._mountable_devices:
             for d in self._mountable_devices:
-                self._container.add(TextWidget(d[1]), callback=self._select_mountable_device, data=d[0])
+                self._container.add(TextWidget(d[1]), callback=self._select_mountable_device,
+                                    data=d[0])
```
Comment (poncovka) [inline pyanaconda/ui/tui/spokes/installation_source.py]: Personally, I don't like this style. I prefer a new line for every argument.

Comment (jkonecny12) [reply pyanaconda/ui/tui/spokes/installation_source.py]: Fair, I'll change that.

**Comment by jkonecny12** [comment]
Will be solved only in F30 branch.

**Comment by rvykydal** [comment]
Applied to master with https://github.com/rhinstaller/anaconda/pull/1843

**Pull request closed without merging**